weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTable.tsx

```diff
@@ -727,36 +841,64 @@ const ExportRunsTableButton = ({
       alignItems: 'center',
     }}>
     <Button
-      className="mx-16"
+      className="mx-4"
       size="medium"
-      variant="secondary"
-      onClick={() => tableRef.current?.exportDataAsCsv()}
-      icon="export-share-upload">
-      Export to CSV
-    </Button>
+      variant="ghost"
+      onClick={onClick}
+      disabled={disabled}
+      tooltip={
+        exportAll ? 'Export table to CSV' : 'Export selected rows to CSV'
+      }
+      icon="export-share-upload"
+    />
   </Box>
 );
 
 const CompareEvaluationsTableButton: FC<{
   onClick: () => void;
   disabled?: boolean;
-}> = ({onClick, disabled}) => (
+  tooltipText?: string;
+}> = ({onClick, disabled, tooltipText}) => (
   <Box
     sx={{
       height: '100%',
       display: 'flex',
       alignItems: 'center',
     }}>
     <Button
-      className="mx-16"
-      style={{
-        marginLeft: '0px',
-      }}
+      className="mr-4"
```

Comment:
some weirdness here because of the icon spacing inside the button I think...

<img width="294" alt="Screenshot 2024-07-10 at 4 27 56 PM" src="https://github.com/wandb/weave/assets/19414170/80080bb1-27c0-472c-befd-610fada7d268">